shell.nix

```diff
@@ -0,0 +1,9 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+(
+  pkgs.buildFHSUserEnv {
+    name = "client-java-shell";
+    targetPkgs = pkgs: with pkgs;[ maven openjdk11 git ];
```

Comment:
currently, client-java is compiled with java8.

Reply:
OK